news/2024.html

```diff
@@ -30,6 +30,52 @@
 </div>
 -->
 
+<!----------------------------------------------------------->
+<div class="news-item tag-link" id="sponsor24-unsw">
+  <a href="#sponsor24-proofcraft"><span class="tag-span"># </span></a> 31 May 2024: Welcome UNSW, Bronze sponsor of the seL4 Summit 2024
+</div>
+<div>
+   <a href="https://unsw.edu.au">
+    <img src="../Foundation/Membership/LOGOS/UNSW.svg"
+      style="width: 15%; float:right"
+      alt="UNSW logo" />
+  </a>
+
+  <p>
+    The seL4 Foundation thanks <a href="https://unsw.edu.au">UNSW</a> for
+    becoming a Bronze sponsor of the <a href="../Foundation/Summit/2024">seL4 Summit
+    2024</a>.
+  </p>
+  <p>
+    seL4 was created by the <a href="https://trustworthy.systems/">Trustworthy Systems (TS)</a> team, which is now part of the <a href="https://unsw.edu.au">University of New South Wales (UNSW)</a>, a founding member of the seL4 Foundation.
+  </p>
+  <p>
+    <a href = "../Foundation/Summit/2024">
+      <img src="../images/summit/sel4_summit_2024.svg"
+      style="width: 15%; float:right; padding-bottom:5px"
+      alt="seL4 summit" />
+    </a>
+    The team uses rigorous formal methods to develop trustworthy software systems—systems that come with provable security, safety and reliability guarantees. Their aims are to:
```

Comment:
The team has a track record of designing an implementing systems software for performance and reliability, and using rigorous formal methods to prove that they meet their security and reliability goals.